chainerx_cc/chainerx/routines/indexing.cc

```diff
@@ -293,4 +296,58 @@ Array Where(const Array& condition, Scalar x, Scalar y) {
     return out;
 }
 
+std::vector<Array> Nonzero(const Array& a) {
+    std::vector<Array> out;
+
+    if (a.ndim() < 1) {
```

Comment:
`nonzero` for 0-d array is deprecated.
https://www.numpy.org/doc/1.17/reference/generated/numpy.nonzero.html

Reply:
Thanks for the information. Have removed it.